File Provider Extension/FileProviderEnumerator.swift

```diff
@@ -213,8 +213,8 @@ class FileProviderEnumerator: NSObject, NSFileProviderEnumerator {
                     if error == .success {
                         DispatchQueue.global().async {
                             NCManageDatabase.shared.convertFilesToMetadatas(files, useMetadataFolder: true) { _, metadatasFolder, metadatas in
-                                let metadatasResult = NCManageDatabase.shared.getMetadatas(predicate: NSPredicate(format: "account == %@ AND serverUrl == %@ AND status == %d", account, serverUrl, NCGlobal.shared.metadataStatusNormal))
-                                NCManageDatabase.shared.updateMetadatas(metadatas, metadatasResult: metadatasResult)
+                                let predicate = NSPredicate(format: "account == %@ AND serverUrl == %@ AND status == %d", account, serverUrl, NCGlobal.shared.metadataStatusNormal)
```

Comment:
I use my own predicates in the new media but I will try to use the internal ones.